lmdeploy/pytorch/engine/engine.py

```diff
@@ -741,6 +748,8 @@ def __update_inputs(next_token_ids):
         logger.debug('<ForwardTask>: '
                      f'batch_size={inputs.seq_length.size(0)} '
                      f'num_tokens={inputs.input_ids.size(-1)}')
+        if self.gpu_count == 1:
```

Comment:
Why do we need this while previously not?

Reply:
logits process can be streaming with cuda inputs.
distribute container with cuda tensor is slower than CPU tensor.